src/mrinufft/io/siemens.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+try:
+    from mapvbvd import mapVBVD
+    MAPVBVD_FOUND = True
+except ImportError:
+    MAPVBVD_FOUND = False
+
+
+def read_rawdat(filename: str, removeOS: bool = False, squeeze: bool = True,
+                data_type: str = "SPARKLING_VE11C"):
+    """Read raw data from a Siemens MRI file.
+    
+    Parameters
+    ----------
+    filename : str
+        The path to the Siemens MRI file.
+    removeOS : bool, optional
+        Whether to remove the oversampling, by default False.
+    squeeze : bool, optional
+        Whether to squeeze the dimensions of the data, by default True.
+    data_type : str, optional
+        The type of data to read, by default 'SPARKLING_VE11C'.
+
+    Returns
+    -------
+    tuple
+        A tuple containing the raw data and the header information.
```

Comment:
describe what the tuple does 
```suggestion
    data: ndarray
        Imported data formatted as XXX 
    hdr: dict
        Extra information about the data parsed from the twix file
```